CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
 # Changelog
 
 ## [unreleased]
+* Fig coloring of ties and slurs (@rettinghaus)
+* Fig bug with tuplet number placement (@rettinghaus)
```

Comment:
Please put bug fixes at the bottom of the list

Reply:
done

Reply:
> Fig coloring/Fig bug

Fix coloring/Fix bug?

Reply:
@craigsapp feel free to improve it.

Reply:
Maybe `Fig` is German for `Fix`? :-)